### PR TITLE
Create ucs_storage_profile.py

### PR DESCRIPTION
**Pull request by sisudhir**

### CODE REVIEW

File: library/ucs_storage_profile.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+DOCUMENTATION = r'''
+---
+module: ucs_storage_profile
+short_description: Creates and deletes storage profile on Cisco UCS Manager
+description:
+- Creates storage profile on Cisco UCS Manager.
+- Examples can be used with the UCS Platform Emulator U(https://communities.cisco.com/ucspe).
+extends_documentation_fragment: ucs
+options:
```
Comment (dsoper2) [inline library/ucs_storage_profile.py]: Please document the other options for the module.  UCS Manager online help provides good descriptions of fields that can be used as is.

File: library/ucs_storage_profile.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        dn_exists = ucs.login_handle.query_dn(dn)
+
+        if module.params['state'] == 'absent':
+            if dn_exists: 
+                # delete mo if dn already exist
+                ucs.login_handle.remove_mo(mo)
+                ucs.login_handle.commit()
+                changed = True               
+        else:
+            if not dn_exists:
```
Comment (dsoper2) [inline library/ucs_storage_profile.py]: See ucs_vlans or other modules for data checking used to determine whether or not a change is needed to a module.  Properties on the managed object should be checked to see if a change is needed.  This module only has description to check, but that should help with the code needed for comparing properties within the modules.

Comment (sisudhir) [reply library/ucs_storage_profile.py]: Yes, David. I'm yet to add the mo property checks. Will do it soon and update the patch. Thank you for reviewing!

**Comment by sisudhir** [comment]
Moved to https://github.com/CiscoUcs/ucsm-ansible/pull/47

**Pull request closed without merging**